scripts/nix/dune.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, ocaml, findlib, opaline, ncurses }:
```

Comment:
Do we still need `dune.nix` file? (I remember Dune was update to version 2.x on Nix packages)

Reply:
Right, good catch, I forgot to remove it, thanks